src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -59,10 +59,12 @@ the MIT License. See LICENSE in the project root for license information. -->
                                 <TextBox x:Uid="Profile_Commandline"
                                          x:Name="Commandline"
                                          Text="{x:Bind State.Profile.Commandline, Mode=TwoWay}"
-                                         Style="{StaticResource TextBoxSettingStyle}"/>
+                                         Style="{StaticResource TextBoxSettingStyle}"
+                                         Visibility="{x:Bind State.Profile.IsBaseLayer, Mode=OneWay, Converter={StaticResource InvertedBooleanToVisibilityConverter}}"/>

```

Comment:
Mind putting this visibility binding on the `ContentPresenter` above?

When we eventually add `SettingsContainer` (a control that'll wrap settings and add inheritance controls [i.e. "inherited from 'Base Layer'", inherit button, etc...]), `ContentPresenter` will be replaced with that `SettingsContainer`. So having the visibility binding on there will make sure the entire thing is removed.

Reply:
Fixed!